### PR TITLE
Add AutoStake standard testnet services

### PR DESCRIPTION
**Pull request by AutoStake-com**

### CODE REVIEW

**Review by ccamel — APPROVED** [review]
Great! Thanks :heart:

File: README.md [hunk 1]
```diff
@@ -141,11 +143,12 @@ src="assets/linktree.webp" style="width:15px;height:15px;"> [Linktree](https://l
 | [NodeStake](https://nodestake.top/)    | <https://nodestake.top/okp4>                       | ✅    |
 | [AVIAONE](https://aviaone.com)         | <https://aviaone.com/blockchains-service/okp4-testnet-okp4-nemeton-1.html>         |   ✅    |
 | [🔥STAVR🔥](https://github.com/obajay)     | <https://github.com/obajay/StateSync-snapshots/tree/main/Projects/OKP4#snapshot-04-gb-updated-every-5-hours>  |   ✅    |
+| [AutoStake](https://autostake.com)    | <https://snapshots.autostake.com/okp4-nemeton-1>   | ✅      |
 
 </details>
 
 <details>
-  <summary>State / Quick Sync</summary>
+  <summ>ary>State / Quick Sync</summary>
```
Comment (amimart) [inline README.md:151]:
```suggestion
  <summary>State / Quick Sync</summary>
```

Can you please revert this change ? :)

**Review by amimart — CHANGES_REQUESTED** [review]
Hey, just a little remark :)

File: README.md (hunk 1 above)
Comment (AutoStake-com) [reply README.md:151]: Interesting thanks, not sure how that happened

**Review by amimart — APPROVED** [review]
Thx 😉

**Review by antho31 — APPROVED** [review]
Thanks!
